lib/SPIRV/libSPIRV/SPIRVModule.cpp

```diff
@@ -1504,8 +1504,11 @@ operator>> (std::istream &I, SPIRVModule &M) {
   Decoder >> MI.InstSchema;
   assert(MI.InstSchema == SPIRVISCH_Default && "Unsupported instruction schema");
 
-  while(Decoder.getWordCountAndOpCode())
-    M.add(Decoder.getEntry());
+  while (Decoder.getWordCountAndOpCode()) {
+    SPIRVEntry *Entry = Decoder.getEntry();
+    if (Entry != NULL)

```

Comment:
It is common to use nullptr instead of NULL here. Please update.
Other than that, LGTM. Thanks.